### DISCUSSION TITLE
How can I download Live Photos from the cloud to iOS? It seems I can only download static images.

### DISCUSSION THREAD

**Discussion by Jason-Young-AI** [immich-app/immich#6807] [Q&A]
I uploaded a Live Photo to the cloud using the Immich iOS app, but it didn't appear to be Live after downloading it. Am I doing something wrong, or is this feature not currently supported?

**Comment by mmomjian** [comment]
I am not able to reproduce this issue. I have pictures that are uploaded on the Immich iOS app and when I download them, I get the image and a video (2 files total). Does your browser allow the downloading of multiple files at once?

**Reply by Jason-Young-AI** [reply]
I uploaded a Live Photo using the Immich iOS app, removed the image from my phone, and then re-downloaded it from the app. However, the image turned into a non-live format. Upon checking the log, I found the following message: "MESSAGE: Asset cannot be saved as a live photo. This is most likely a motion photo. Saving only the image file. FROM: ImageViewerService."

When I attempted to log in to Immich via a mobile browser and save the image, only the HEIC part of the Live Photo could be downloaded, and the MOV format prompted an error: "Error downloading IMG_xxx.mov."

**Comment by 1298345095** [comment]
Me too. And it's even not a hdr photo.
